src/components/Layout/index.jsx

```diff
@@ -83,14 +84,15 @@ class Layout extends Component {
       title,
       isHome,
       match,
+      noSubMenu,
       pageTitle
     } = this.props;
 
     const { isNormal, isDev, isAdmin } = this.props.user;
     const hasMenu = (isDev || isAdmin) && !isHome;
     const paths = ['/dashboard', '/profile', '/ssh_keys', '/dev/apps'];
     const isCenterPage = Boolean(pageTitle); // detail page, only one level menu
-    const hasSubNav = hasMenu && !isCenterPage && !paths.includes(match.path);
+    const hasSubNav = !noSubMenu && hasMenu && !isCenterPage && !paths.includes(match.path);
```

Comment:
`hasMenu` should check before `noSubMenu`:
```
 hasMenu && !noSubMenu
```

Reply:
ok